packages/big-design-patterns/src/components/Page/styled.tsx

```diff
@@ -32,6 +33,15 @@ export const StyledPageBackground = styled.div.attrs({ theme: defaultTheme })<{
       `;
     }
   }}
+
+  ${({ actionBar }) => {
+    if (actionBar) {
+      return css`
+        padding-block-end: ${({ theme }) => theme.helpers.remCalc(48)}};
+      `;
+    }
+  }}
```

Comment:
Instead of passing in props, you can reference other styled components:
```suggestion
  &:has(${StyledActionBar}) {
    padding-block-end: ${({ theme }) => theme.helpers.remCalc(48)}};
  }
```

```tsx
import { StyledActionBar } from '../ActionBar/styled';
```

Reply:
added, thanks @chanceaclark